exhibitions/app/controllers.js

```diff
@@ -18,7 +20,7 @@ export async function renderExhibition(ctx, next) {
     if (format === 'json') {
       ctx.body = exhibitionContent;
     } else {
-      ctx.render('exhibition', {
+      ctx.render('pages/exhibition', {
```

Comment:
Baring in mind that this was a naming strategy for one big mega app - we don't have to be replicating things, and should be using this opportunity to fo and learn rolling forward...

Not saying this shouldn't be called this, but I wonder, would a service have more than 1 page - and is this really a page, or just a very detailed exhibition-list component? I would ask the conceptual question of what is the difference between a page and component?

Reply:
For me, conceptually, I think a page would be routable.